frontend/src/javascripts/components/users/header.js

```diff
@@ -4,25 +4,75 @@ import { Link } from 'react-router-dom'
 import ImgHistoryIcon from '../../../images/main/history_icon.png'
 import ImgSettingIcon from '../../../images/main/setting_icon.png'
 import ImgBackIcon from '../../../images/main/back_icon.png'
+import ImgUser from '../../../images/main/user_default_icon.png'
 
```

Comment:
思ったけど、この辺は`constants.js`に集約しなくていい？

Reply:
どこに移すの？

Reply:
`constants.js`に。
　
```suggestion
import { ImgHistoryIcon, ImgSettingIcon, ImgBackIcon, ImgUser } from '../../constants';
```
みたいな

Reply:
チェゴ👍
それでお願いします！

Reply:
あ、俺がやるのか

Reply:
いや、僕やります！

Reply:
お願いします！
直接iomportしてるのはtoppage.js, header.js, mypage.jsにもあるから、全部const経由するようにしてあげると後々非常に管理しやすいと思う！